x/slashing/client/cli/query.go

```diff
@@ -57,7 +58,7 @@ func GetCmdQueryParams(cdc *codec.Codec) *cobra.Command {
 		Args:  cobra.NoArgs,
 		Long: strings.TrimSpace(`Query genesis parameters for the slashing module:
 
-$ <appcli> query slashing params
+` + fmt.Sprintf("$ %v query slashing params", version.ClientName) + `
```

Comment:
Instead of the funky:

```
+ `
`),
```

Why not just pass `\n` to `fmt.Sprintf`?

Reply:
Well I used macros to resolve all of these (so this was the common solution) also I actually think it looks a bit better (code is less cramped after trying both ways) - but honestly don't matter to me, if you have a preference I can switch all instances of this to use `\n`

Reply:
```go
	return &cobra.Command{
		Long: strings.TrimSpace(`Query genesis parameters for the slashing module:

Example:
` + fmt.Sprintf("$ %v query slashing params\n", version.ClientName),
		),
```

Reads better than:

```go
	return &cobra.Command{
		Long: strings.TrimSpace(`Query genesis parameters for the slashing module:

` + fmt.Sprintf("$ %v query slashing params", version.ClientName) + `
`),
```

I don't mind changing it. Thoughts?

Reply:
sounds good to me! - if you feel like making the change be my guest - if not I'll do it in a few hours or tomorrow